--- root/Makefile~	2009-03-18 15:43:56.000000000 +0100
+++ root/Makefile	2009-03-18 15:42:56.000000000 +0100
@@ -963,17 +963,6 @@
 	   $(INSTALLDATA) fonts/*               $(DESTDIR)$(TTFFONTDIR); \
 	   find $(DESTDIR)$(TTFFONTDIR) -name CVS -exec rm -rf {} \; >/dev/null 2>&1; \
 	   find $(DESTDIR)$(TTFFONTDIR) -name .svn -exec rm -rf {} \; >/dev/null 2>&1; \
-	   echo "Installing misc docs in $(DESTDIR)$(DOCDIR)"; \
-	   $(INSTALLDIR)                        $(DESTDIR)$(DOCDIR); \
-	   $(INSTALLDATA) LICENSE               $(DESTDIR)$(DOCDIR); \
-	   $(INSTALLDATA) README/*              $(DESTDIR)$(DOCDIR); \
-	   find $(DESTDIR)$(DOCDIR) -name CVS -exec rm -rf {} \; >/dev/null 2>&1; \
-	   find $(DESTDIR)$(DOCDIR) -name .svn -exec rm -rf {} \; >/dev/null 2>&1; \
-	   echo "Installing tutorials in $(DESTDIR)$(TUTDIR)"; \
-	   $(INSTALLDIR)                        $(DESTDIR)$(TUTDIR); \
-	   $(INSTALLDATA) tutorials/*           $(DESTDIR)$(TUTDIR); \
-	   find $(DESTDIR)$(TUTDIR) -name CVS -exec rm -rf {} \; >/dev/null 2>&1; \
-	   find $(DESTDIR)$(TUTDIR) -name .svn -exec rm -rf {} \; >/dev/null 2>&1; \
 	   echo "Installing tests in $(DESTDIR)$(TESTDIR)"; \
 	   $(INSTALLDIR)                        $(DESTDIR)$(TESTDIR); \
 	   $(INSTALLDATA) test/*                $(DESTDIR)$(TESTDIR); \
